frontend/src/container/PublicProfile.jsx

```diff
@@ -626,5 +526,5 @@ export class PublicProfile extends React.Component {
   }
 }
 
-const mapStateToProps = ({ breadcrumbs, user }) => ({ breadcrumbs, user })
+const mapStateToProps = ({ breadcrumbs, user, tlm }) => ({ breadcrumbs, user, tlm })
```

Comment:
We have a tlm object on redux?

Reply:
Actually, I'm not sure if this is required here. We used it for the setHeadTitle if I remember correctly